src/GitLink/Extensions/ProjectExtensions.cs

```diff
@@ -94,10 +76,9 @@ public static string GetOutputPdbFile(this Project project)
         {
             Argument.IsNotNull(() => project);
 
-            var outputFile = project.GetOutputFile();
-            var pdbFile = Path.ChangeExtension(outputFile, ".pdb");
+            var outputFile = project.GetPropertyValue("PdbFile");
```

Comment:
I haven't looked at VS2013 recently. But I've been on the VS project & build team since VS2010, and we never rename properties because that breaks compatibility. And PDBs existed back then, so I'm pretty certain this property has been named the same since MSBuild was created. :) I'd be happy to respond to a bug if it ever came in that this is wrong.
One case where it might be wrong is not "back in VS2013" as much as "for vcxproj". vcxproj does a lot of things differently than csproj/vbproj. But then, I couldn't get gitlink to work with vcxproj originally anyway, so there was nothing to maintain compatibility with.